### DISCUSSION TITLE
Conditional Flex?

### DISCUSSION THREAD

**Discussion by lbssousa** [chakra-ui/chakra-ui#4393] [Q&A]
What's the equivalent, in Chakra UI components, to the following Tailwind CSS styling?

```html
<div class="lg:flex lg:items-start">
  ...
</div>
```
I mean, a component that just enables flexbox under certain breakpoints

**Comment by aaronkik** [comment]
You'll want to look at this: https://chakra-ui.com/docs/features/responsive-styles

An example for you would be
```javascript
<Box display={{base: "block", lg:"flex"}}>
...
</Box>
```

**Reply by eric-hc** [reply]
Do you know if it is possible to use responsive styles with custom props? For example,

```jsx
<Trending full={{sm: false, lg: true}} />
```

So that it would set `full` to either `true` or `false` based on the breakpoint(s)?

**Reply by aaronkik** [reply]
Sure, you could use this for non Chakra components: https://chakra-ui.com/docs/hooks/use-breakpoint-value 

Maybe in this example you showed me the hook would be: 

```javascript
const isFull = useBreakpointValue({ base: false, lg: true })

<Trending full={isFull} />
```

**Comment by anubra266** [comment]
```html
<Box display={{ lg: "flex" }} alignItems={{ lg: "start" }}>
...
</Box>
```
